cSpell.json

```diff
@@ -0,0 +1,46 @@
+{
+	"version": "0.2",
+	"language": "en",
+	"words": [
+		"ADAL",
+		"Authenticode",
+		"Blazor",
+		"blockquotes",
+		"Bootstrapper",
+		"Contoso",
+		"Cupertino",
+		"Dismissable",
+		"Docfx",
+		"emscripten",
+		"Flatpickr",
+		"Infragistics",
+		"layouting",
+		"Listview",
+		"Liu's",
+		"MSAL",
+		"msbuild",
+		"MVVM",
+		"netstd",
+		"oidc",
+		"overscroll",
+		"Silverlight",
+		"Skia",
+		"skiasharp",
+		"slnf",
+		"struct",
+		"Syncfusion",
+		"Uno's",
+		"UWP's",
+		"VSIX"
+	],
+	"ignoreWords": [],
+	"ignoreRegExpList": [
+		"\\((.*)\\)", // Markdown links
+		"```[a-z]*\n[\\s\\S]*?\n```", // Markdown code blocks. h/t https://coderwall.com/p/r6b4xg/regex-to-match-github-s-markdown-code-blocks,
```

Comment:
Can you use the regex specified [here](https://cspell.org/configuration/patterns/) instead? It's a more accurate regex.

Reply:
Done! Also due to your tip I learned about cSpell's ability to name patterns and then ignore based on name, which I like and may use elsewhere.